.github/workflows/build-push-docker-image.yml

```diff
@@ -112,11 +112,6 @@ jobs:
           # cache-from: type=gha
           # cache-to: type=gha,mode=max
 
```

Comment:
Removing tests execution fastens up the image generation process but how do release owner know if the test cases are indeed successful before going with the release ?
Is this not the common .yaml being used across the environments ? Ideally we need skip of test cases execution only in hotfix workflow.

Reply:
> Is this not the common .yaml being used across the environments ? Ideally we need skip of test cases execution only in hotfix workflow.

I was thinking there is no need to execute the tests in docker image build process.

>how do release owner know if the test cases are indeed successful before going with the release ?

A developer can check for DT Tests workflow for seeing what happened with the tests. And this workflow executes for every PR & it can be checked in the PR itself.

Reply:
Tests are also running in `test_and_upload` workflow which can be checked for checking what has happened with tests